### PR TITLE
#1625 Turkish translation 

### PR DESCRIPTION
**Pull request by demirhalil**
#1625 

Pull request description

- Turkish translation is added for singleton design pattern

### CODE REVIEW

File: tr/singleton/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+```java
+var enumIvoryTower1 = EnumIvoryTower.INSTANCE;
+var enumIvoryTower2 = EnumIvoryTower.INSTANCE;
+assertEquals(enumIvoryTower1, enumIvoryTower2); // true
+```
+
+## Sınıf diagramı
+
+![alt text](./etc/singleton.urm.png "Singleton pattern class diagram")
```
Comment (ohbus) [inline tr/singleton/README.md:57]: Need to change this to the original file.

**Review by ohbus — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please resolve the conflicts.

**Comment by demirhalil** [comment]
I will close the Pr because I forgot to update the recent changes from master and after merging it in my local, it shows a lot of changes. So will open a new pr for this.

**Comment by ohbus** [comment]
You do not need to close it. You could have just rebased and force push.

**Comment by demirhalil** [comment]
It is my first the first open source project that I contribute which is why I did not want to do something wrong. If I have again such situation again, I'll do what you said. Thank you for the suggestion :))

**Pull request closed without merging**